### ISSUE TITLE
Allow users to display up to 5 items as a table on their profile

### ISSUE THREAD

**Issue by Sylvhem** [florence-social/mastodon-fork#43]
Mastodon allow users to add up to four items on their profile that will be displayed as a table. Based on my observations, those items are often used for indicating:
- pronouns;
- languages;
- secondary account;
- website;
- donation profile (Liberation, Patreon, Ko-fi, etc.);
- instant messaging handle (Matrix, Discord, XMPP, etc.).

Obviously, not everyone is using all those things. But we can see how having only four items can be inconvenient. I have been struggling with that myself.
If it doesn't bring any trouble from a design perspective, I think we should allow users to display up to five items, not four. It'll give them a little more flexibility and has the advantage to feeling a little less like a random choice.

**Comment by Feufochmar** [comment]
Why five, and why not more ? I don't see why it should be limited to so few items, at least in theory (I'm pretty sure people will ask for more than five). However, there are UI considerations before adding more items, notably in the column showing profiles.

There is already a problem of vertical space when looking at a profile when there are items in the table. Adding more will worsen things. On smaller screens, the first toots may already not be visible on the column without scrolling.

I displayed your profile in a column, took a screenshot and made a mock-up of what it could look like if there was 5 items in the table. In height, I have on my screen 980 pixels for displaying a web page (100px are took by the desktop and web browser for displaying various bars). 

![sylvhem-comparison-4vs5](https://user-images.githubusercontent.com/8533643/53301149-7485ac00-384f-11e9-93c3-7d0d700bdffa.png)

A bio like yours using 4 items tooks a bit more than half of the available space (510 px). Add the buttons related to the column UI (50px for "go back", 170px for the various things displayed below the bio but above the toots) and the small space above the columns. It takes 735px (+100px, so 835px) before starting to show the toots of the person.

Since the scrollbar is not clearly visible on the default theme, a new user on a small screen may be unaware that more content is available below a profile.

**Comment by Sylvhem** [comment]
> Why five, and why not more ?

Because adding more will likely cause UI problems, like you pointed out.
But, in the vacuum, I'm not against letting people add more that five (let's say ten). If people want to clutter their profile, why not? But I think five seems like a reasonable choice.

**Comment by ghost** [comment]
My .02

> Since the scrollbar is not clearly visible on the default theme, a new user on a small screen may be unaware that more content is available below a profile.

I would rather address the scrollbar visibility and allow more / arbitrary / admin configable number of things in your bio table.  I don't think [users mind scrolling](https://uxmyths.com/post/654047943/myth-people-dont-scroll)

**Comment by clarfonthey** [comment]
The number allowed should probably be configurable.
